.github/workflows/wolfssl.yml

```diff
@@ -35,7 +35,7 @@ jobs:
        tar -xzf v5.0.0-stable.tar.gz
        cd wolfssl-5.0.0-stable
        ./autogen.sh
-       ./configure --enable-tls13 --enable-all --enable-harden --prefix=$HOME/wssl
+       ./configure --enable-tls13 --enable-opensslextra --enable-harden --prefix=$HOME/wssl
```

Comment:
I don't think you should remove the old `--enable-all` build but I think adding an `--enable-opensslextra` is a good ideas. Note that the second build is a *lesser* wolfSSL build and for example NTLM will not be enabled in your curl build using that.

Reply:
I have had a go at splitting the wolfssl CI job in two to test the two configurations.